lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -27,7 +27,7 @@ def run!(args)
     end
 
     def analyze
-      @matches = @output.chomp.split("\n\n").map { |m| m.split("\n") }
+      @matches = @output.chomp.split("\n\n").map { |m| m.gsub(/\n\s+\[/, "SPLIT_ME_HERE_PLEASE[").split("SPLIT_ME_HERE_PLEASE") }
```

Comment:
Yes, splitting on a new line doesn't work if there is an embedded new line.  In this case you end up with two smells, and there was a specific check in the test to ignore the second one.  There's this comment in the test for that:

  > # Note: hopefully a temporary solution until I figure out how to deal with newlines in the 
  >             method call more effectively -Jake 5/11/2009